src/MspFcComms.cpp

```diff
@@ -60,6 +60,16 @@ namespace FcComms
     }
 
     // Send the rc commands to the FC using the member array of rc values.
+    FcCommsReturns MspFcComms::getRawRC()
```

Comment:
You have to pass the array in here by reference or a pointer. Preferably by reference. Then the values will be returned to the callee.